### DISCUSSION TITLE
When is the next release?

### DISCUSSION THREAD

**Discussion by nathanbirrell** [Shopify/prettier-plugin-liquid#44] [Q&A] [answered]
Thanks for the hard work on this plugin.

Wondering,hen is the next release? Keen to make use of this fix! https://github.com/Shopify/prettier-plugin-liquid/issues/29

**Comment by charlespwd** [comment, marked as answer]
Was planning on making one once #41 is in. So probably next week!

**Reply by charlespwd** [reply]
Released!
